package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "presentationtools-aps",
-  "version": "1.0.0",
-  "api_version": "1.0.0",
+  "version": "2.0.0",
+  "api_version": "2.0.0",
```

Comment:
`api_version` is a companion thing and needs to remain at 1.0.0.  for `version`, this feels more like a 1.1.0 to me.  Thoughts?

Reply:
1.1.0 for the version would be better. I will change.